client/src/lib/convert.js

```diff
@@ -1,5 +1,12 @@
-export const base64ToByteArray = string =>
-  Uint8Array.from(atob(string), c => c.charCodeAt(0));
+export const base64ToByteArray = string => {
+  // String replace because server with encode with 'web safe' base64_encode
+  const b = atob(string.replace(/_/g, '/').replace(/-/g, '+'));
+  return Uint8Array.from(b, c => c.charCodeAt(0));
+};
 
-export const byteArrayToBase64 = byteArray =>
-  btoa(String.fromCharCode(...(new Uint8Array(byteArray))));
+export const byteArrayToBase64 = byteArray => {
+  // We specifically do not want to make the 'web safe' string replacements above
+  // doing so will break this functionality
+  const uarr = new Uint8Array(byteArray);
+  return btoa(String.fromCharCode(...uarr));
+};
```

Comment:
Looks like two method that are pretty easy to test. Maybe write some test for them if times allows.

Or even better, find a well supported third party lib to do this and use that instead.

Reply:
Added tests